distribution/test/itests/test-itests-catalog/src/test/java/ddf/catalog/test/AbstractIntegrationTest.java

```diff
@@ -166,11 +172,33 @@ public void initFacades() {
      */
     @org.ops4j.pax.exam.Configuration
     public Option[] config() throws URISyntaxException {
+        basePort = findPortNumber(20000);
+
+        HTTP_PORT = String.valueOf(basePort + 1);
+        HTTPS_PORT = String.valueOf(basePort + 2);
+        SSH_PORT = String.valueOf(basePort + 3);
+        RMI_SERVER_PORT = String.valueOf(basePort + 4);
+        RMI_REG_PORT = String.valueOf(basePort + 5);
+        LOGGER.debug(HTTP_PORT + " " + HTTPS_PORT + " " + SSH_PORT + " " + RMI_SERVER_PORT + " "
```

Comment:
Adding labels to the log would be helpful. Otherwise it's just going to be a bunch of numbers with no context.